### ISSUE TITLE
Invalid Version: 0.4.0rc7

### ISSUE THREAD

**Issue by ex1st** [raineorshine/npm-check-updates#1129]
- [x ] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [ x] I am using the latest version of `npm-check-updates`
- [ x] I am using `node >= 14`

---

## Steps to Reproduce
add [grunt-contrib-requirejs](https://www.npmjs.com/package/grunt-contrib-requirejs) to dependencies
ncu --target=minor
## Current Behavior
```
Checking /home/user/project/package.json
[--------------------] 0/1 0%/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/build/src/index.js:26
    throw err;
    ^

TypeError: Invalid Version: 0.4.0rc7
    at new SemVer (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/node_modules/semver/classes/semver.js:38:13)
    at compare (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/node_modules/semver/functions/compare.js:3:3)
    at Object.gt (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/node_modules/semver/functions/gt.js:2:29)
    at compareVersions (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/build/src/version-util.js:171:29)
    at Array.sort (<anonymous>)
    at Object.findGreatestByLevel (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/build/src/version-util.js:188:10)
    at minor (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/build/src/package-managers/npm.js:359:24)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getPackageVersionProtected (/home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/build/src/lib/queryVersions.js:69:30)
    at async /home/user/.nvm/versions/node/v16.15.0/lib/node_modules/npm-check-updates/node_modules/p-map/index.js:57:22
```

**Comment by raineorshine** [comment]
`0.4.0rc7` is not a valid semver version. npm-check-updates only works with valid semver versions. 

I recommend adding `grunt-contrib-requirejs` to the reject list in a `.ncurc.json` file. 

``` 
{
  "reject": ["grunt-contrib-requirejs"]
}
```

**Comment by raineorshine** [comment]
Fixed and published in `v13.1.1`
